CSS/Default.sublime-keymap

```diff
@@ -56,7 +70,7 @@
 	// Expand {|} to { | } when space is pressed
 	{ "keys": [" "], "command": "insert_snippet", "args": {"contents": " $0 "}, "context":
 		[
-			{ "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
+			{ "key": "setting.smart_typing", "operator": "equal", "operand": true },
```

Comment:
What is the rationale for gating this behind `smart_typing` only? Shouldn't it also require `auto_match_enabled`?

Reply:
Probably due to the assumption that someone who enables `smart_typing` would not disable `auto_match_enabled`.

The setting is just an attempt to help those who want more control over when space is added or not. I am pretty heppy with this binding in general and use it in every auto-match-scenario. 

I'd be ok with keeping it gated by `auto_match_enabled` only.